src/payments/payments.controller.ts

```
@@ -51,6 +51,8 @@ export class PaymentsController {
   })
   @ApiBadRequestResponse({ description: 'Invalid payment data' })
   async initiatePayment(@Body() dto: InitiatePaymentDto) {
+    console.log('initiatePayment-----------------------controller called');
```

ESLint

[error] Replace `'initiatePayment-----------------------controller·called'` with `"initiatePayment-----------------------controller·called"`

(prettier/prettier)